pkg/kubelet/metrics/metrics.go

```diff
@@ -108,6 +108,10 @@ const (
 	CPUManagerPinningRequestsTotalKey = "cpu_manager_pinning_requests_total"
 	CPUManagerPinningErrorsTotalKey   = "cpu_manager_pinning_errors_total"
 
+	// Metrics to track the Memory manager behavior
```

Comment:
Can you update the KEP with these metrics?

Reply:
Thank you @kannon92 , there was a mistake in the KEP.
I've changed all the instances but missed one. Now it should be fine.